context/app/static/js/shared-styles/sections/LabelledSectionText/LabelledSectionText.jsx

```diff
@@ -19,7 +19,9 @@ function LabelledSectionText({ children, label, iconTooltipText, bottomSpacing,
           </SecondaryBackgroundTooltip>
         )}
       </Flex>
-      <Typography variant="body1">{children}</Typography>
+      <Typography component={childContainerComponent} variant="body1">
```

Comment:
If undefined defaults to a `<p>`?

Reply:
That was the default behavior I was observing, which led to errors where it complained about `ul` elements inside of `p` elements; I will explicitly set this to `p` by default in the props to avoid any future confusion or regressions from upstream changes to this default.